docs/tutorials/react-native.mdx

```diff
@@ -0,0 +1,708 @@
+peer token. The `connect` function establishes a connection with the Jellyfish server
+via web socket and authenticates the peer.
+
+```tsx title="/screens/Connect.tsx"
+// highlight-next-line
+import { useJellyfishClient } from "@jellyfish-dev/react-native-client-sdk";
+import { NavigationProp } from "@react-navigation/native";
+
+interface ConnectScreenProps {
+  navigation: NavigationProp<any>;
+}
+
+// This is the address of the Jellyfish backend. Change the local IP to yours. We
+// strongly recommend setting this as an environment variable, we hardcoded it here
+// for simplicity.
+// highlight-next-line
+const JELLYFISH_URL = "ws://X.X.X.X:4000/socket/peer/websocket";
+
+function ConnectScreen({ navigation }: ConnectScreenProps): JSX.Element {
+  const [peerToken, setPeerToken] = useState<string>("");
+
+  // highlight-next-line
+  const { connect } = useJellyfishClient();
+
+  // highlight-start
+  const connectToRoom = async () => {
+    try {
+      await connect(JELLYFISH_URL, peerToken.trim());
+    } catch (e) {
+      console.log("Error while connecting", e);
+    }
+  };
+  // highlight-end
+  return (
+    <View style={styles.container}>
+      <TextInput
+        placeholder="Enter peer token"
+        value={peerToken}
+        onChangeText={setPeerToken}
+      />
+      // highlight-next-line
+      <Button onPress={connectToRoom} title="Connect" disabled={!peerToken} />
+      <QRCodeScanner onCodeScanned={setPeerToken} />
+    </View>
+  );
+}
+
+// ...
+```
+
+### Camera permissions (Android only)
+
+To start the camera we need to ask the user for permission first. We'll use a
+standard React Native module for this:
+
+```tsx title="/screens/Connect.tsx"
+import { 
+  View,
+  StyleSheet,
+  type Permission,
+  PermissionsAndroid,
+  Platform, 
+ } from "react-native";
+
+// ...
+
+function ConnectScreen({ navigation }: ConnectScreenProps): JSX.Element {
+  // ...
+
+  // highlight-start
+  const grantedCameraPermissions = async () => {
+    if (Platform.OS === 'ios') return true;
+    const granted = await PermissionsAndroid.request(
+      PermissionsAndroid.PERMISSIONS.CAMERA as Permission
+    );
+    if (granted !== PermissionsAndroid.RESULTS.GRANTED) {
+      console.error("Camera permission denied");
+      return false;
+    }
+    return true;
+  };
+  // highlight-end
+
+  const connectToRoom = async () => {
+    try {
+      await connect(JELLYFISH_URL, peerToken.trim());
+
+      // highlight-start
+      if (!(await grantedCameraPermissions())) {
+        return;
+      }
+      // highlight-end
+    } catch (e) {
+      console.log("Error while connecting", e);
+    }
+  };
+
+  // ...
+}
+
+// ...
+```
+
+### Starting the camera
+
+Jellyfish Client provides a handy hook for managing the camera: `useCamera`. It
+can not only start a camera but also toggle, switch between multiple cameras,
+manage camera state and camera track simulcast settings and bandwidth. Also when
```

Comment:
```suggestion
Not only it can start a camera but also toggle it, manage its state, simulcast and bandwidth settings, and switch between multiple sources.
```